api/plugin_support.h

```diff
@@ -108,6 +108,8 @@ typedef enum {
   BRANCH_CALL = (1 << 9),
   BRANCH_INTERWORKING = (1 << 10), // A32 and T32
   BRANCH_TABLE = (1 << 11),        // T32-only
+  RISCV_BRANCH = (1 << 12), // RISCV only
+  RISCV_JUMP = (1 << 13), //RISCV only
```

Comment:
The only ISA-specific types we define here and use in `mambo_get_branch_type()` are for properties which can't be represented with the generic flags. I don't think this applies to `RISCV_BRANCH` and `RISCV_JUMP`. Unless I'm missing something, please remove them.

Reply:
@lgeek My thinking here was, since a branch on arm can refer to something that is conditional or unconditional, I thought it would be nice to be able to make the distinction for RISC-V, since branches on RISC-V are conditional and jumps are the unconditional counterpart.